### PR TITLE
https://issues.redhat.com/browse/ACM-14574--new function on install disc

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

**Review by dislbenn — APPROVED** [review]
LGTM

File: install/install_disconnected.adoc [hunk 1]
```diff
@@ -243,3 +243,29 @@ spec: {}
 The `mce-subscription-spec` annotation is required because {mce-short} is automatically installed during the {acm-short} installation. If you are creating the resource with a CLI, include the `mce-subscription-spec` annotation in the YAML that you apply with the `oc apply` command to create the `MulticlusterHub` resource.
 
 If you create the resource by using the OperatorHub console, switch to the _YAML view_ and insert the annotation as previously displayed. *Important:* There is no field in the OperatorHub console for the annotation in the _Field view_ panel to create the `MulticlusterHub` resource.
+
+[#disconnect-catalog-priority]
+=== Catalog source priority
+
+When the `MulticlusterHub` resource prepares to install the {mce-short}, it implements `CatalogSource` priority as criteria.
```
Comment (dockerymick) [inline install/install_disconnected.adoc]: Should this be `MultiClusterHub`? When I searched through the doc, I saw both instances being used. We can create a tech debt issue to make sure we are referencing the correct resource name.

File: install/install_disconnected.adoc [hunk 2]
```diff
@@ -243,3 +243,29 @@ spec: {}
...
+
+The {acm-short} `MulticlusterHub` resource seeks the `CatalogSource` that contains the desired {mce-short} version that is compatible with the current {acm-short} version. 
+
+If there are multiple `CatalogSource` resources available, the `MulticlusterHub` resource selects the catalog source with the highest `spec.priority` value that is set within the resource instances. 
+
+If a custom `CatalogSource` is created without a priority level, is set to `0` and used as the target `CatalogSource`.
```
Comment (dockerymick) [inline install/install_disconnected.adoc]:
```suggestion
If a custom `CatalogSource` is created without a priority level, it is set to `0` and used as the target `CatalogSource`.
```

**Review by dockerymick — APPROVED** [review]
Two comments. Otherwise, this looks good to me

File: install/install_disconnected.adoc (hunk 1 above)
Comment (swopebe) [reply install/install_disconnected.adoc]: Yes, you are right. I can change for this and get to the rest another time.

Comment (swopebe) [reply install/install_disconnected.adoc]: all have been changed

**Pull request merged**